packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -67,6 +73,23 @@ class OpPerfTelemetry {
             this.firstConnection = false;
         });
 
+        this.deltaManager.outbound.on("push", (messages) => {
+            for (const msg of messages) {
+                if (this.clientSequenceNumberForLatencyStatistics === msg.clientSequenceNumber) {
+                     this.opTimeSittingInInboundQueue = Date.now();
+                }
```

Comment:
I like defensive programming. It has a con of potentially screwing container if we get logic wrong, but once proven by testing and time it ensures stronger invariants are in place and system is more reliable. Or it will help us understand right in the moment that we did not took care of some case.

Here, I'd assert that this.opTimeSittingInInboundQueue is undefined when you get there. Same for this.opTimeSittingInInboundQueue.